src/sync/sync_session.cpp

```diff
@@ -461,7 +461,7 @@ void SyncSession::handle_error(SyncError error)
             // Connection level errors
             case ProtocolError::connection_closed:
             case ProtocolError::other_error:
-            case ProtocolError::pong_timeout:
+//            case ProtocolError::pong_timeout:
```

Comment:
Yes, that entire commit is just to make it compile with a not-yet-released version of sync, and adapting to the changes there once it's actually released should be done separately from this PR.